src/Expression/ExpressionDescription.php

```diff
@@ -4,7 +4,7 @@
 
 namespace Arkitect\Expression;
 
-class Description
+class ExpressionDescription
```

Comment:
Why is it better this name for you instead of `Description`? 
Having already `Expression` in the namespace it sounds like a repetition or not?

Reply:
I renamed it since otherwise there'd be a `Description` and a `ViolationDescription`

Maybe I should move `ViolationDescription` to be in `Arkitect\Rules`? (next to `Violation`) Then I could revert this to be Description without any confusion

Reply:
I think that we can do it in another PR after this one because it can have a different scope. At the moment it's good enough this name.